File: mindflow/utils/response.py
"""
Handle response text from Mindflow API.
"""
# import pyperclip


def handle_response_text(text: str, skip_clipboard: bool = False):
    """
    Copy to clipboard or print response text.
    """
    # if not skip_clipboard:
    #     try:
    #         pyperclip.copy(f"\n{text}")
    #         print("Response copied to clipboard!!!")
    #         return
    #     except Exception:
    #         # raise Exception(f"Failed to copy to clipboard: {error}")
    #         pass
    print(text)

File: mindflow/src/utils/response.rs
use clipboard::{ClipboardContext, ClipboardProvider};

pub fn handle_response_text(text: String, skip_clipboard: bool) {
    if !skip_clipboard {
        let mut ctx: ClipboardContext = ClipboardProvider::new().unwrap_or_else(|e| {
            panic!("Failed to create clipboard context: {}", e);
        });
        ctx.set_contents(text).unwrap_or_else(|e| {
            panic!("Failed to copy to clipboard: {}", e);
        });
        println!("Response copied to clipboard!!!");
        return
    }
    println!("{}", text);
}